mods/player_api/api.lua

```diff
@@ -135,3 +145,43 @@ minetest.register_globalstep(function(dtime)
 		end
 	end
 end)
+
+-- player get node vars
+local update_time = 1.0
+local timer = 0
+
+minetest.register_globalstep(function(dtime)
+
+	timer = timer + dtime
+	if timer < update_time then
+		return
+	end
+	timer = 0
+
+	local name, pos, ndef
+
+	for _,player in pairs(minetest.get_connected_players()) do
+
+		name = player:get_player_name()
+		pos = player:getpos()
+
+		pos.y = pos.y - 0.1
+		player_api.nodes[name].stand = minetest.get_node_or_nil(pos) or
+				minetest.registered_nodes["air"]
```

Comment:
Do you mean
```Lua
stand = minetest.registered_nodes[minetest.get_node_or_nil(pos) or "air"]
```
?

Reply:
nope, if I do that then param2 is lost from the actual node whn storing, it only resorts to "air" if it cannot get a node from the map (e.g. player moving too fast during map update)

Reply:
Then why do you fall back on the air *definition*? That's inconsistent.